templates/docs/patterns/hero/index.md

```diff
@@ -0,0 +1,102 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Hero | Patterns
+---
+
+A hero is a prominent banner section typically used to quickly capture the user's attention after they land on the page.
+Depending on the size and composition of your content, you can choose from a variety of hero layouts.
+
+The hero is constructed using the following inputs:
+
+| Input                                                      | Description                                                                                                                                                                            |
+| ---------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <code>layout</code> (**required**)                         | One of: <ul><li><code>50/50</code></li><li><code>50/50-full-width-image</code></li><li><code>25/75</code></li><li><code>75/25</code></li><li><code>fallback</code> (default)</li></ul> |
+| <code>title</code>                                         | Hero title text, displayed as H1                                                                                                                                                       |
+| <code>subtitle</code>                                      | Hero subtitle text, displayed as H2                                                                                                                                                    |
+| <code>is_split_on_medium</code>                            | Whether the layout is split on medium in 50/50, 25/75, and 75/25 layouts                                                                                                               |
+| <code>description</code>                                   | Free paragraph text                                                                                                                                                                    |
+| <code>cta</code>                                           | Call to action block                                                                                                                                                                   |
+| <code>image</code>                                         | Primary hero image                                                                                                                                                                     |
+| <code>signpost_image</code> (**required on 25/75 layout**) | Signpost image                                                                                                                                                                         |
```

Comment:
I don't think it's required. We have tons of examples (in design guidelines page) of 25/75 heros without signpost.

I know whey were not part of current spec in Figma, but they do exist. So let's not document it as required.